src/Php/TypeAnalyzer.php

```diff
@@ -8,7 +8,11 @@
 {
     public function isPhpReservedType(string $type): bool
     {
-        return in_array($type, ['string', 'bool', 'mixed', 'object', 'iterable', 'array', 'float', 'int'], true);
+        return in_array(
+            $type,
+            ['string', 'bool', 'null', 'false', 'true', 'mixed', 'object', 'iterable', 'array', 'float', 'int'],
```

Comment:
Isn't boolean missing here as it can be used as phpdoc value as well as bool?

Reply:
Maybe. Where should be add test to cover it and be sure?

Reply:
I guess for uses cases you use in code, it won't fail.